data/schema.yaml

```diff
@@ -221,6 +242,7 @@ components:
           - $ref: '#/components/schemas/subheadline'
           - $ref: '#/components/schemas/text'
           - $ref: '#/components/schemas/video'
+          - $ref: '#/components/schemas/aresMedia'
```

Comment:
Can we change this to just say `media`? I think adding `ares` might confuse things and `media` is generic enough.

Reply:
"Ideally Ares shouldn’t be creating `media` blocks unless we know that Optimo won’t also in the future create `media` blocks. So we currently decided to namespace them with who is responsible for doing it.
There is a conversation around a system whereby other people can submit new block types to Optimo without creating name clashes to hopefully resolve this"

Reply:
Thanks for the clarification! PR looks good to me :)